packages/material-ui/src/Breadcrumbs/Breadcrumbs.test.js

```diff
@@ -45,4 +47,52 @@ describe('<Breadcrumbs />', () => {
     assert.strictEqual(wrapper.is('.test-class-name'), true);
     assert.strictEqual(wrapper.hasClass(classes.root), true);
   });
+
+  it('should render seperators', () => {
+    const wrapper = shallow(
+      <Breadcrumbs>
+        <span />
+        <span />
+        <span />
+      </Breadcrumbs>,
+    );
+    assert.strictEqual(wrapper.find(BreadcrumbSeparator).length, 2);
```

Comment:
Maybe mount and test that it properly intersperses? `containsMatchingElement` is a good candidate.

Reply:
I'm passing on this one. I can't find an easy way to do it. We have the visual regression test to help.

Reply:
I had some issues with containsMatchingElement as well. I hoped that you were able to fix it so I can copy the solution :smile:

Reply:
Shame on me